src/dbms/inmemory/replication_handlers.cpp

```diff
@@ -1046,6 +1046,33 @@ uint64_t InMemoryReplicationHandlers::ReadAndApplyDeltas(storage::InMemoryStorag
             throw utils::BasicException("Failed to drop point index on :{}({})", data.label, data.property);
           }
         },
+        [&](WalVectorIndexCreate const &data) {
+          spdlog::trace("       Create vector index on :{}({})", data.label, data.property);
+          auto *transaction = get_replication_accessor(delta_timestamp, kUniqueAccess);
+          auto labelId = storage->NameToLabel(data.label);
+          auto propId = storage->NameToProperty(data.property);
+          auto const unum_metric_kind =
+              unum::usearch::metric_from_name(data.metric_kind.data(), data.metric_kind.size());
+          if (unum_metric_kind.error) {
+            throw utils::BasicException("Failed to create vector index on :{}({}), invalid metric kind: {}", data.label,
+                                        data.property, data.metric_kind);
```

Comment:
"... supported metric kinds are X, Y, Z." NOTE: If more is listed here, applications like GraphChat could use that to generate the right query. It's ok to add the link but "AI Agents" are not efficient in reading links.